sdk/src/main/java/io/customer/sdk/CustomerIO.kt

```diff
@@ -9,9 +9,50 @@ import io.customer.sdk.data.model.CustomAttributes
 import io.customer.sdk.data.communication.CustomerIOUrlHandler
 import io.customer.sdk.data.model.Region
 import io.customer.sdk.data.request.MetricEvent
-import io.customer.sdk.data.store.CustomerIOStore
 import io.customer.sdk.di.CustomerIOComponent
 
+interface CustomerIOInstance {
```

Comment:
The interface is created to allow customers to mock our SDK. Since `CustomerIO` class is `final`, I created this interface to allow mocking using Mockito. 

I'll add a comment to the interface to explain this use case. That is the only reason the interface was created.

Reply:
Complete.